requirements/voice_client.txt

```diff
@@ -0,0 +1,8 @@
+neon-utils~=1.8,>=1.8.3a3
+ovos-dinkum-listener~=0.0.2,>=0.0.3a27
+ovos-vad-plugin-webrtcvad~=0.0.1
+ovos-microphone-plugin-alsa~=0.0.0
```

Comment:
Consider sounddevice for wider cross-platform support

Reply:
Whoops, didn't mean to include the voice client in this PR. Still WIP, so I'll add that note but remove the added files from this branch